tests/test_rescan.py

```diff
@@ -193,7 +197,6 @@ def test_rescan_blended(reqctx, fake_github, fake_jira):
     assert "changed" not in ret
 
     # We shouldn't have made any writes to GitHub or Jira.
-    fake_github.assert_readonly()
```

Comment:
Rescans of blended PRs are no longer guaranteed to be read-only, as we may be `POST`ing to the status endpoint, even if the other state is unchanged.

I'm _assuming_ this is a reasonable change of expectations, but open to having missed what this is actually checking, so let me know...